qiita_pet/static/js/sharing.js

```diff
@@ -0,0 +1,56 @@
+var current_study = null;
```

Comment:
Is there any way to not use this global variable? Global variables in JavaScript are a slippery slope, and it tends to be really hard to recover from its (ab)use.